src/openrct2/config/Config.cpp

```diff
@@ -615,6 +615,7 @@ namespace Config
         log_verbose("config_find_rct1_path(...)");
 
         static constexpr const utf8* searchLocations[] = {
+            R"(/usr/share/openrct2)",
```

Comment:
I'm pretty sure this is not correct. These are locations for `RollerCoaster Tycoon Deluxe`, part 1 of the series.

Reply:
A question: Is is possible to install both data files, from RCT1 and RCT2 at the same time?
Or only one at a time?

Reply:
They serve different purposes. The RCT2 files are needed for the game, while the RCT1 files are optional and provide some extra graphics. So a user will either have RCT2 installed, or RCT2 _and_ RCT1.

Reply:
Ah! Allright, that clarifies it for me.